pages/lib/global_scheme.py

```diff
@@ -112,13 +112,21 @@
 thermal_stress_label = "Thermal stress"
 
 mapping_dictionary = {
-    "None": {"name": "None"},
-    "DOY": {"name": "Day of the year", "unit": "days", "range": [0, 365]},
-    "day": {"name": "day", "range": [1, 31]},
-    "month": {"name": "months", "unit": "months", "range": [1, 12]},
-    "hour": {
-        "name": "Hour",
-        "color": [
+    ColNames.NONE: {ColNames.NAME: "None"},
```

Comment:
for the moment this is okay but it is very confusionary, please let me explain how we can fix this in the next meeting, basically we do not need both ColNames and this mapping dictionary, instead we should have the following:

```python
class VariablesInfo
      name: str
      unit: str
      .....

class ColNames:
    # ==================== Time related column ====================
    DAY = VariablesInfo(
name = "day of the Year"
unit="day"
col_name='day"
....)
```

do not implement this now. I opened an issue so we can fix it next week.